lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -728,6 +740,8 @@ def test_gradebook(self):
         )
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
+        # Max number of student per page is one.  Patched setting MAX_STUDENTS_PER_PAGE_GRADE_BOOK = 1
+        self.assertEqual(len(response.mako_context['students']), 1)  # pylint: disable=no-member
```

Comment:
still getting issue on jenkins

lms/djangoapps/ccx/tests/test_views.py (94.4%):
11:49:10     744: E1101: (no-member), TestCCXGrades.test_gradebook: Instance of 'WSGIRequest' has no 'mako_context' member